### PR TITLE
key-wrapper: Add test for chained command and fix bug

### PR DESCRIPTION
**Pull request by hanazuki**
@pyama86 

Always terminate key list returned from stns-key-wrapper by a newline.
This ensures keys from STNS and chained command are separated by a NL.

### CODE REVIEW

File: cmd/key/stns-key-wrapper_test.go [hunk 1]
```diff
@@ -13,12 +14,18 @@ type Response struct {
 	path, query, contenttype, body string
 }
 
+func useTestBins(t *testing.T) func() {
+	origPath := os.Getenv("PATH")
+	os.Setenv("PATH", "./test-fixtures/bin:/bin:/usr/bin")
+	return func() { os.Setenv("PATH", origPath) }
+}
```
Comment (udzura) [inline cmd/key/stns-key-wrapper_test.go:21]: Looks cool

File: cmd/key/test-fixtures/bin/get-external-keys [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+if test "$#" -ne 1 || test -z "$1" ; then
```
Comment (pyama86) [inline cmd/key/test-fixtures/bin/get-external-keys:3]: useful!!

**Comment by pyama86** [comment]
LGTM!
Thanks in advance!

**Comment by hanazuki** [comment]
I've merged master

**Comment by pyama86** [comment]
Thanks!

**Pull request merged**